metrics-proxy/src/main/java/ai/vespa/metricsproxy/service/NodeMetricGatherer.java

```diff
@@ -0,0 +1,210 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.metricsproxy.service;
```

Comment:
I suggest to move this to a new 'node' package, as it's not related to Vespa services.